popup/popup.js

```diff
@@ -116,3 +118,57 @@ $(collapseBtn).on("click", function() {
     expandBtn.style.display = "flex";
     expandSection.style.display = "none";
 })
+
+downloadButton.onclick = function() {
+    const data = {};
+    try {
+        browser.storage.sync.get(null)
+        .then(items => {
+            const entries = Object.entries(items);
+            entries.forEach(entry => {
+                data[entry[0]] = entry[1].url;
+            });
+            save(data);
+        })
+    } catch (e) {
+        alert("Failed! Error: " + e.toString());
+    }
+};
+
+function save(data) {
```

Comment:
Avoid naming function with too little wording. What are you trying to save here?

Reply:
Right, I'm trying to save all the golinks. I just changed the function name to exportGoLinks.